src/EditorFeatures/VisualBasicTest/Formatting/Indentation/SmartIndenterTests.vb

```diff
@@ -2835,6 +2835,44 @@ End Class"
                 expectedIndentation:=12)
         End Sub
 
+        <WorkItem(3293, "https://github.com/dotnet/roslyn/issues/3293")>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.SmartIndent)>
+        Public Sub TestSmartIndentInArgumentLists3()
+            Dim code = "
+Class C
+    Sub M()
+        Console.WriteLine(""{0} + {1}"",
+            19, 23 +
```

Comment:
:memo: This test doesn't cover the case I was describing because `19` is an argument that starts a line. `23 +` gets excluded because it is on the same line as 19. I'm looking for a test case where an argument meets *both* of the following conditions:

1. The argument is the first argument that starts on the line
2. The first token of the argument is not the first token on the line

Reply:
Sorry, I missed the first `+` in your original example. Will update.

Reply:
And FWIW, it will certainly use the indentation of the first argument in that case. However, I think I'm comfortable with that.

Reply:
No problem, the additional ones you did add were good tests too

Reply:
And FWIW, I would expect that it would indent to the second argument in this case. I don't think that's necessarily correct, but it's a pretty fishy scenario.